bootstrap-salt.sh

```diff
@@ -4474,7 +4485,7 @@ install_openbsd_deps() {
             # Let's download, since they were not provided, the default configuration files
             if [ ! -f "$_SALT_ETC_DIR/$fname" ] && [ ! -f "$_TEMP_CONFIG_DIR/$fname" ]; then
                 ftp -o "$_TEMP_CONFIG_DIR/$fname" \
-                    "https://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
+                    "${HTTP_VAL}://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
```

Comment:
I that really necessary for GitHub? Anyhow you'll get HTTP 301 code with `https://` URL.

```console
$ curl -v http://raw.githubusercontent.com/saltstack/salt/develop/conf/master
*   Trying 23.235.39.133...
* Connected to raw.githubusercontent.com (23.235.39.133) port 80 (#0)
> GET /saltstack/salt/develop/conf/master HTTP/1.1
> Host: raw.githubusercontent.com
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Server: Varnish
< Retry-After: 0
< Location: https://raw.githubusercontent.com/saltstack/salt/develop/conf/master
< Content-Length: 0
< Accept-Ranges: bytes
< Date: Tue, 12 Apr 2016 11:55:43 GMT
< Via: 1.1 varnish
< Connection: close
< X-Served-By: cache-atl6231-ATL
< X-Cache: HIT
< X-Cache-Hits: 0
< Access-Control-Allow-Origin: *
< Expires: Tue, 12 Apr 2016 12:00:43 GMT
< Vary: Authorization,Accept-Encoding
< 
* Closing connection 0
```